docs/Configuration.md

```diff
@@ -190,6 +192,47 @@ trigger:
   limit: 10 mb
 ```
 
+For `time`, it has three field, _interval_, _modulate_ and _max_random_delay_.
+
+The _interval_ field is a string which defines the time to roll the
+file. The interval field supports the following units(second will be used if the
+unit is not specified), case does not matter:
+
+- second[s]
+- minute[s]
+- hour[s]
+- day[s]
+- week[s]
+- month[s]
+- year[s]
+
+> Note: `log4j` set `Sunday` as the first day of the week, but `log4rs` set
+> `Monday` as the first day of the week, which is the same as `chrono` crate
+> and the `ISO 8601` standard. So if you use `week`, the log file will be rolled
+> on `Monday` instead of `Sunday`.
```

Comment:
```suggestion
> Note: `log4j` treats `Sunday` as the first day of the week, but `log4rs` treats
> `Monday` as the first day of the week, which is the follows the `chrono` crate
> and the `ISO 8601` standard. So when using `week`, the log file will be rolled
> on `Monday` instead of `Sunday`.
```